src/types.ts

```diff
@@ -27,10 +28,17 @@ export type GetSortedNodes = (
         | 'importOrder'
         | 'importOrderBuiltinModulesToTop'
         | 'importOrderCaseInsensitive'
-        | 'importOrderSeparation'
         | 'importOrderGroupNamespaceSpecifiers'
+        | 'importOrderMergeDuplicateImports'
+        | 'importOrderSeparation'
         | 'importOrderSortSpecifiers'
     >,
 ) => ImportOrLine[];
 
 export type GetChunkTypeOfNode = (node: ImportDeclaration) => string;
+
+export type GetImportFlavorOfNode = (node: ImportDeclaration) => string;
```

Comment:
This type is only used in the function declaration itself.  We can remove this, and specify the types inline at the function declaration itself.

Reply:
I guess there are a lot of those in this file, we can clean that up later.  But I think that `string` could be a bit more specific, and we could use the union of the values we actually expect to get, here.

Reply:
I was only trying to match the patterns I saw already in the codebase. But happy to make the return type more explicit!

Doing that in a commit now. -- Technically this is a typescript-breaking change but not a runtime change.

Reply:
Yep, I'm pretty new to actually working in this code myself, so some of the patterns that exist here are a little odd.  Not blaming you, but any means.  :-D. Best to stay consistent for now, and then cleanup later if we decide to.